apps/02RockPaperScissors.js

```diff
@@ -4,13 +4,47 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
+var hand1Wins = "Hand one wins!";
+var hand2Wins = "Hand two wins!"
 function rockPaperScissors(hand1, hand2) {
 
     // Write code here
+    // If this statement is true, the code reader quits reading the code.
+    if (hand1 === hand2) {
+      return "It's a tie!"
+    }
+
+    if (hand1 === "rock") {
```

Comment:
This is a little verbose, but it shows that you definitely understand the logic. I like it! This is how I wrote it too :D